e2e/app/element/base-element.ts

```diff
@@ -132,7 +133,7 @@ export default class BaseElement {
         .catch(() => {
           return null;
         });
-      return (await jsHandle.jsonValue()) as boolean;
+      return await jsHandle.jsonValue();
```

Comment:
I'm not sure it made sense in the original, but this `await` doesn't make sense here. Could perhaps do a separate PR to keep the linter's changes pure.
```suggestion
      return jsHandle.jsonValue();
```